_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,441 @@
+
+- This rule assumes that the language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+The rule doesn't require the non-visual characteristic description to be included in the accessibility tree. If the alternatives are not included in the accessibility tree, only [Success Criteria 1.3.1 Info and Relationships][sc131] would fail instead of both [Success Criterion 1.3.3 Sensory Characteristics][sc133] and [Success Criteria 1.3.1 Info and Relationships][sc131]. Hence, the rule passes in these cases as it is not a failure of all accessibility requirements.
+
+The identified web content does not have to be positioned on the same web page and doesn't need to be linked to from the tested web page.
+
+- [WCAG 2.1 - Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [WCAG 2.1 - Understanding Success Criterion 1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+- [G96: Providing textual identification of items that otherwise rely only on sensory information to be understood](https://www.w3.org/WAI/WCAG21/Techniques/general/G96)
+- [F14: Failure of Success Criterion 1.3.3 due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+- [F26: Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information](https://www.w3.org/WAI/WCAG21/Techniques/failures/F26)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition.
+
+```html
+<head>
+	<title>Passed example 1 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+This paragraph includes the [visual reference word][] "box". The button is identified with the word "box" but also identified by referencing that the content can be found below this content in the DOM order. Note that "below" is also a visual reference word but in this case it can also be accurately interpreted as "further in the DOM tree order" which does not rely on visual attributes alone. Hence, the paragraph matches the (**non-visual reference**) condition.
+
+```html
+<p>Interact with the box below this paragraph, for a surprise</p>
```

Comment:
Hey Adilofficial,

Thanks for the detailed reply.

The word "below" **is** a visual reference word, that is why it is included in the list of visual reference words. However in this case it can **also** be interpreted as referring to "further in the reading order". 

It is true that the example gives no definition of this term and that it might still be misunderstood by users. However the SC only asks us to not "rely solely on sensory characteristics" which we do not because in this case we are also relying on the DOM/reading order. The SC doesn't demand that this is explained with a definition.

This example would of course not be a best practice and we should never advise people to do this. But technically it would pass the SC, and such edge cases should be made clear by the rule. Would you agree with me that the example should pass the SC?

You are right however that this example would not currently pass this rule. I think the rule needs another expectation:

- (**word-is-used-non-visually**) the test target identifies some [web content][], but at least one of the visual reference words that identifies the web content is used in such a way that it can be correctly interpreted non-visually.

Reply:
Thus said, I'm also considering whether this example is actually showing two things and should be split:
* "box" as a correct-ish word to describe a button, could be "click on the box after this paragraph" to show the same.
* "below" as a visual reference word with a non-visual reference meaning, could be "click on the button below" to show the same.

Also, for the **word is used non visually**, a less controversial example could be "click on the button right after this paragraph" where "right" is a visual reference word used to give indication, but not with a sensory meaning.

At the same time, I agree with @ShadowBB that our examples should rather show the corner cases for the sake of reproducibility and unification of tests results.